npm-shrinkwrap.json

```diff
@@ -1738,7 +1738,24 @@
       }
     },
     "interpolate-components": {
-      "version": "1.0.4"
+      "version": "1.0.4",
```

Comment:
Needs `i18n-calypso` to be bumped first -- https://github.com/Automattic/i18n-calypso/issues/3